core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +60,30 @@ def map(exploration_model):
     def reduce(exp_id, list_of_exps):
         pass
 
+class ExpSummariesContributorsOneOffJob(jobs.BaseMapReduceJobManager):
+    """One-off job that finds the user id's of the contributors
+    (defined as any human who has made a 'positive' -- i.e.
+    non-revert-- commit) for each exploration.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationSnapshotMetadataModel]
+
+    @staticmethod
+    def map(item):
+        _COMMIT_TYPE_REVERT = 'revert'
+        if (item.commit_type != _COMMIT_TYPE_REVERT and
```

Comment:
Indentation should be as follows:

```
        if (item.commit_type != _COMMIT_TYPE_REVERT and
                item.committer_id not in feconf.SYSTEM_USER_IDS):
            exp_id = item.get_unversioned_instance_id()
            yield (exp_id, item.committer_id)
```

The last two lines form the inner block and are indented by 4 from the 'if'. The continuation from the if is indented a bit more to distinguish it from those lines.